cmd/kpromo/cmd/pr/pr.go

```diff
@@ -316,33 +355,35 @@ func runPromote(opts *promoteOptions) error {
 	}
 
 	// Push to fork
-	if mustRun(opts, fmt.Sprintf("Push changes to user's fork at %s/%s?", userForkOrg, userForkRepo)) {
-		logrus.Infof("Pushing manifest changes to %s/%s", userForkOrg, userForkRepo)
+	if mustRun(opts, fmt.Sprintf("Push changes to user's fork at %s/%s?", opts.userFork, k8sioRepo)) {
+		logrus.Infof("Pushing manifest changes to %s/%s", opts.userFork, k8sioRepo)
 		if err := repo.PushToRemote(github.UserForkName, branchname); err != nil {
-			return fmt.Errorf("pushing %s to %s/%s: %w", github.UserForkName, userForkOrg, userForkRepo, err)
+			return fmt.Errorf("pushing %s to %s/%s: %w", github.UserForkName, opts.userFork, k8sioRepo, err)
 		}
 	} else {
 		// Exit if no push was made
-
-		logrus.Infof("Exiting without creating a PR since changes were not pushed to %s/%s", userForkOrg, userForkRepo)
+		logrus.Infof("Exiting without creating a PR since changes were not pushed to %s/%s", opts.userFork, k8sioRepo)
 		return nil
```

Comment:
The tool is supposed to exit here, however, it would continue running after this function. You should do something similar as in `handleManifests` or use `os.Exit`.